### ISSUE TITLE
How to use the node?

### ISSUE THREAD

**Issue by snot1988** [mcnaveen/n8n-nodes-randomizer#2]
How should I use this node? 

I think the logic should be another way... I send the data to the node and so this node would randomize to A or B... no?

**Comment by mcnaveen** [comment]
Yeah, That's right. You have to go to settings and select "Community Nodes"

Give this community node name "n8n-nodes-randomizer" and install it.

![9B1XTaNuvThxFq6V000109](https://github.com/user-attachments/assets/c3c33642-62e5-4b6c-9bca-20c1364aff57)

You can find this node

![xypBZso2AtC3Avba000110](https://github.com/user-attachments/assets/5b38dc82-35c8-422f-b7ef-b199f3af2f38)

Then in the canvas use this node

![kzFn8VIyytSWKtdd000111](https://github.com/user-attachments/assets/d31d2572-3bc7-41f7-a0d8-b8c055689a80)
